src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -219,6 +236,10 @@ const en = ({ isDivorce, partner, userCase }: CommonContent, alternativeServiceT
       link: '/downloads/bailiff-service',
     },
   },
+  subHeading1:
+    userCase.state === 'AwaitingAmendedApplication'
```

Comment:
use State.AwaitingAmendedApplication rather than 'AwaitingAmendedApplication' - it's more future-proof